cookbooks/fb_consul/recipes/default.rb

```diff
@@ -0,0 +1,157 @@
+#
+# Cookbook:: fb_consul
+# Recipe:: default
+#
+# Copyright (c) 2020-present, Vicarious, Inc.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+node.default['fb_iptables']['filter']['INPUT']['rules']['consul']['rules'] = [
+  # Gossip protocol between agents and servers
+  '-p tcp --dport 8301 -j ACCEPT',
+  '-p udp --dport 8301 -j ACCEPT',
+]
+
+node.default['fb_users']['users']['consul'] = {
+  'comment' => 'consul agent user',
+  'home' => '/run/consul',
+  'shell' => '/usr/sbin/nologin',
+}
+
+package 'consul' do
```

Comment:
add the `manage_packages` knob here for consistency

Reply:
will do